test/resources/microshift-process.resource

```diff
@@ -76,6 +76,12 @@ Wait For MicroShift Service
     Wait Until Keyword Succeeds    50x    10s
     ...    MicroShift Service Is Active
 
+Wait For MicroShift Service Restart
```

Comment:
This may be too-specific to the test you're writing to be reusable. It assumes something else is causing MicroShift to restart, right?

Reply:
microshift service  is restarted after time change [detected](https://github.com/openshift/microshift/blob/main/pkg/sysconfwatch/sysconfwatch_linux.go#L139),
i was assuming we will need this keyword for diferent cases (IP change?)

Reply:
OK, reusing it for IP change tests makes sense.